packages/showcases/css/stories/components/card/examples/full-image.html

```diff
@@ -16,7 +16,7 @@
   </div>
 </div>
 
-<div class="block">
+<div class="block" style="width: 50%">
   <div class="vtmn-card vtmn-card_variant--full-image">
     <div class="vtmn-card_image">
       <img
```

Comment:
There's a problem with the second full image in dark mode:
![Capture d’écran 2021-11-30 à 23 13 56](https://user-images.githubusercontent.com/9600228/144136850-fa98649e-879c-4683-bb24-d3202341e388.png)